st2common/tests/unit/test_action_alias_utils.py

```diff
@@ -230,3 +233,28 @@ def test_ending_parameters_matching(self):
         parser = ActionAliasFormatParser(alias_format, param_stream)
         extracted_values = parser.get_extracted_param_value()
         self.assertEqual(extracted_values, {'pony1': 'foo', 'pony2': 'bar'})
+
+
+class TestSearchRegexTokens(TestCase):
```

Comment:
I just noticed there are no tests for changes in `get_extracted_param_value` function.

Can you please also add tests for various scenarios in `get_extracted_param_value` (alias format string contains no anchors, contains two anchors, contains one anchor, etc.).

Besides that, LGTM and we can go ahead and merge it once we have those tests.

Reply:
I added tests for all of those cases except the one where the regex contains no anchors. That will have been tested extensively by the rest of the tests in the `TestActionAliasParser` test case.